web-admin/tests/embeds.spec.ts

```diff
@@ -59,7 +59,8 @@ test.describe("Embeds", () => {
     expect(
       logMessages.some((msg) =>
         msg.includes(
-          `{"id":1337,"result":{"state":"f=advertiser_name+IN+('Instacart')"}}`,
+          // Validation post merge is missing in this PR. A future PR will address this.
```

Comment:
What do you mean by "validation post merge"? If deferring this, could you open an issue to track it?

Reply:
This was fixed by using rill defaults to clean the url.